packages/package_info_plus_web/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: package_info_plus_web
 description: Web platform implementation of package_info_plus
 homepage: https://github.com/fluttercommunity/plus_plugins/tree/main/packages/package_info_plus_web
-version: 0.1.0
+version: 0.1.1
```

Comment:
As explained before, we will take care of updating the package version number when releasing.